onnx_tf/frontend.py

```diff
@@ -197,20 +196,20 @@ def tensorflow_graph_to_onnx_graph(cls,
 
         versions = frontend_tf_opset_version[op_name_to_lower(node.op)]
 
+        opset_ver = opset[""]
```

Comment:
Potentially there are multiple domains right, so should we iterate through the keys?

Reply:
This PR is just a hot fix.
Multiple domains support need more discussion.
The main reason is I am not sure onnx supports multiple domain.
https://github.com/onnx/onnx/blob/master/docs/IR.md#model-metadata
I will comment on issue.

Reply:
Maybe I should not add dict interface({'': 6, 'ai.onnx.ml':1}) and keep int in this PR.